app_legacy/Helpers/database/release.php

```diff
@@ -110,7 +110,12 @@ function getIntegrationRelease(): ?array
 
 function getReleasesFromFile(): ?array
 {
-    return file_exists(storage_path('app/releases.php')) ? require_once storage_path('app/releases.php') : null;
+    try {
+        return file_exists(storage_path('app/releases.php')) ? require_once storage_path('app/releases.php') : null;
+    } catch (Throwable) {
```

Comment:
Do we really want to just swallow this exception? Shouldn't it at least get logged?

It tripped me up for about 15 minutes because the `use` path in my `releases.php` hadn't been updated and things relying on the releases information were just misbehaving.

Reply:
Also, as updating that is a manual step, we must remember to do it when we deploy, and need to somehow inform all devs after this gets merged.

Reply:
Very good point. Sorry that I missed that in the self-review - I wanted to add a comment about that initially.

See https://github.com/RetroAchievements/RAWeb/pull/1285/commits/c0ef78bcfdbba642de5ef4edf3e84e7be07c2d96